src/components/Navigation/components/Item/Item.tsx

```diff
@@ -55,7 +55,7 @@ export interface ItemProps extends ItemURLDetails {
   new?: boolean;
   subNavigationItems?: SubNavigationItem[];
   secondaryAction?: SecondaryAction;
-  onClick?(): void;
+  onClick?(event: MouseEvent<HTMLElement>): void;
```

Comment:
Adding the same `event` parameter to `ItemProps.onClick` here for parity with `SubNavigationItem.onClick`.